apps/web/src/components/pages/HomePage/sections/Events.tsx

```diff
@@ -635,17 +634,28 @@ const Events: React.FC<IProps> = props => {
             </FilterWrapper>
             <EventsWrapper>
               {makeEvents()}
-              <h1 style={{ display: (countEvents() === 0 ? '' : 'none') }}>No events found with the given filter.</h1>
+              <h1 style={{ display: countEvents() === 0 ? "" : "none" }}>
+                No events found with the given filter.
+              </h1>
             </EventsWrapper>
           </div>
-          <button
-            style={{ display: countEvents() <= 3 ? "none" : "" }}
-            onClick={toggleNumEvents}
+          <Link
+            style={{ margin: "0 auto" }}
+            to={
+              countEvents() >= 3 && maxEvents !== events.events.length
+                ? ""
+                : "events"
+            }
           >
-            {countEvents() >= 3 && maxEvents !== events.events.length
-              ? "SHOW ALL EVENTS"
-              : "SHOW FEWER EVENTS"}
-          </button>
+            <button
+              style={{ display: countEvents() <= 3 ? "none" : "" }}
```

Comment:
can we move `countEvents() <= 3` into a const and change `3` to like `MAX_EVENTS_TO_DISPLAY`